web/profiles/joinup/joinup.behat.inc

```diff
@@ -2562,4 +2562,28 @@ JS;
     $last->setValue($value);
   }
 
+  /**
+   * Checks if the output is exactly the same as the contents of a given file.
+   *
+   * @param string $file_path
+   *   The path to the file.
+   *
+   * @throws \Exception
+   *
+   * @Then the output should match the file contents of :file
+   */
+  public function theOutputShouldMatchTheFile($file_path) {
+    if ($this->getMinkParameter('files_path')) {
+      $fullPath = rtrim(realpath($this->getMinkParameter('files_path')), DIRECTORY_SEPARATOR) . DIRECTORY_SEPARATOR . $file_path;
+      if (is_file($fullPath)) {
+        $file_path = $fullPath;
+      }
+    }
+    $expected_content = file_get_contents($file_path);
+    $browser_content = $this->getSession()->getPage()->getContent();
```

Comment:
Let's `trim()` both, `$expected_content` and `$browser_content`, to ensure that a trailing new line at the end of the file will not affect the assertion. For the sake of coding standards I would add the new line at the end of the fixture files.